### PR TITLE
Dom task

### PR DESCRIPTION
**Pull request by nvklap**
### Dom api task

[Demo](https://nvklap.github.io/dom-prjct/) |
[Code base](https://github.com/nvklap/dom-prjct)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by nvklap** [comment]
I've completed self-checks.

**Comment by lysenko-sergey-developer** [comment]
Awesome work 🚀

**Review by lysenko-sergey-developer — CHANGES_REQUESTED** [review]

**Comment by lysenko-sergey-developer** [comment]
UI/UX Improvement:
Content reflow while image loading
[The Complete Guide to Lazy Loading Images | CSS-Tricks](https://css-tricks.com/the-complete-guide-to-lazy-loading-images/#aa-tip-3-avoid-content-reflow)
- change `max-width` to `width` on img
- add `aspect-ratio` with a value which your image has
Screenshot 2022-09-13 at 19 41 58

**Comment by nvklap** [comment]
Thanks a lot for review and useful advise! I've fixed my code.

File: submissions/nvklap/dom/js/app.js [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+
+  if (navList.className.includes('show')) {
+    toggleMenu();
+  }
+}
+
+addNavigation(taskmasterData);
+initContent(contentWrapper, initTaskmasterData);
+
+const navList = document.querySelector('.series-list');
```
Comment (lysenko-sergey-developer) [inline submissions/nvklap/dom/js/app.js:405]: Move `navList` to other selectors.

File: submissions/nvklap/dom/js/app.js [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+  parent.innerHTML = '';
+  parent.append(...template);
+}
+
+function initContent(parent, initSeriesData) {
+  const htmlTemplate = createContent(initSeriesData);
+  addTemplateToDom(parent, htmlTemplate);
+}
+
+function showContent({ target }) {
```
Comment (lysenko-sergey-developer) [inline submissions/nvklap/dom/js/app.js:382]: `addNavigation`, `createContentList` and `showContent` do too much stuff. Try to decompose them into smaller functions.

**Review by lysenko-sergey-developer — CHANGES_REQUESTED** [review]

**Comment by lysenko-sergey-developer** [comment]
LGTM! 😃

**Pull request merged**